recipe-server/normandy/conftest.py

```diff
@@ -39,4 +40,14 @@ def fake_sign(datas):
         return sigs
 
     mocked.return_value.sign_data.side_effect = fake_sign
+
+    def verify_api_pair(recipe_and_signature):
```

Comment:
Nit: `verify_signed_api_response(signed_api_response)` is a more appropriate name for this.